project/Dependencies.scala

```diff
@@ -47,15 +47,20 @@ object Dependencies {
 
   val akkaVersion = "2.5.3"
   val akkaStreams = Seq("com.typesafe.akka" %% "akka-stream" % akkaVersion)
-  val akkaHttp = Seq("com.typesafe.akka" %% "akka-http" % "10.0.8")
+  val akkaHttp = Seq("com.typesafe.akka" %% "akka-http" % "10.0.11")
+  val akkaHttpCore = Seq("com.typesafe.akka" %% "akka-http" % "10.0.11")
```

Comment:
Is this duplication intentional? (`akkaHttp` is used, at least once, in [`build.sbt`](https://github.com/2m/play-ws/blob/146d19b456b34b9fa9cc110f97bc77639217b085/build.sbt#L419) but not in `Dependencies`)

If the duplication is intentional, what is the difference between `akkaHttp` and `akkaHttpCore`?

Reply:
Sorry, messed this up. My idea was to make the new akka-http backend depend only on akka-http-core. But later I needed more (marshalling) from akka-http. Fixed in https://github.com/playframework/play-ws/pull/205/commits/108ee306a72343dadd1634fe2f5cca280bede025